src/components/App/App.tsx

```diff
@@ -1,13 +1,49 @@
-import React from 'react';
-import { TestComponent } from '../../TestComponent';
+import React, { useState, useEffect } from 'react';
+// import { TestComponent } from '../../TestComponent';
 import Dashboard from '../pages/Dashboard/Dashboard';
+import EditPage from '../pages/EditPage/EditPage';
+import { Route, Switch } from 'react-router-dom';
+import { ThisIsUser, UserData } from '../../types'
+
+interface DBProps {
```

Comment:
We could possibly move this into the types file for better organization, once we feel confident with how typescript operates.